kamelet-support/src/test/java/io/kaoto/backend/metadata/parser/step/kamelet/KameletParseCatalogTest.java

```diff
@@ -30,6 +39,11 @@
 @WithKubernetesTestServer
 class KameletParseCatalogTest {
 
+    @ConfigProperty(name = "GITHUB_TOKEN")
+    private Optional<String> githubToken;
```

Comment:
Uh, does this mean the tests won't run on my local machine?

Reply:
I guess not, I see a proper handling below.

Reply:
No. When the token is not provided and is empty, the Authorization header [is not set](https://github.com/KaotoIO/kaoto-backend/blob/01ff5718fa3ba19d9387025e2d738c30c29587e9/kamelet-support/src/test/java/io/kaoto/backend/metadata/parser/step/kamelet/KameletParseCatalogTest.java#L183) so if your machine has not exceeded the Github API limits, the test will work.